src/System.Runtime/tests/System/GCTests.cs

```diff
@@ -112,12 +119,17 @@ public static void KeepAlive()
 
         private class KeepAliveTest
         {
+            [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
+            private static void MakeAndDropDNKA()
+            {
+                new DoNotKeepAliveObject();
```

Comment:
If there is no behavioral difference between the 2 patterns (I hope there's none, but one never knows for sure), I don't care that much which pattern you use or if it is a bit inconsistent. I was just curious if there is more to it :)